tests/Admin/ArticleAdminTest.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Tests\AppBundle\Admin;
+
+use AppBundle\DataFixtures\ORM\LoadAdminData;
+use AppBundle\DataFixtures\ORM\LoadArticleData;
+use AppBundle\Entity\Article;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Tests\AppBundle\Controller\ControllerTestTrait;
+use Tests\AppBundle\MysqlWebTestCase;
+
+class ArticleAdminTest extends MysqlWebTestCase
+{
+    use ControllerTestTrait;
+
+    public function testCreateCategoryFail(): void
+    {
+        $this->authenticateAsAdmin($this->client, 'superadmin@en-marche-dev.fr', 'superadmin');
+
+        $crawler = $this->client->request('GET', $this->getUrl('admin_app_article_create'));
```

Comment:
Please use `Symfony\Component\HttpFoundation\Request::METHOD_GET`
(and same for all such calls below)